shallowblue_rllib.py

```diff
@@ -379,8 +391,9 @@ def get_observation(self, world_state):
                 elif yaw == 90:
                     obs = np.rot90(obs, k=3, axes=(1, 2))
 
-                allow_break_action = observations['LineOfSight']['type'] == 'diamond_ore' or \
-                    observations['LineOfSight']['type'] == 'coal_ore'
+                allow_break_action = (observations['LineOfSight']['type'] == 'diamond_ore' or \
+                    observations['LineOfSight']['type'] == 'coal_ore') 
+                    # and (obs[1, int(self.obs_size/2)-1, int(self.obs_size/2)] == 1)
```

Comment:
i fee like this 'and' condition is needed cuz the agent starts mining while its floating down or when other times when its too far from the ore to actually mine anything. But when I added the condition it didn't seem to mine at all so maybe one of y'all can fix this?